bookops_worldcat/metadata_api.py

```diff
@@ -185,41 +191,34 @@ def get_full_bib(
 
         return query.response
 
-    def holding_get_status(
+    def get_institution_holdings(
         self,
-        oclcNumber: Union[int, str],
-        inst: Optional[str] = None,
-        instSymbol: Optional[str] = None,
-        response_format: Optional[str] = "application/atom+json",
+        oclcNumbers: Union[str, List[Union[str, int]]],
         hooks: Optional[Dict[str, Callable]] = None,
     ) -> Optional[Response]:
         """
         Retrieves Worlcat holdings status of a record with provided OCLC number.
         The service automatically recognizes institution based on the issued access
         token.
-        Uses /ih/checkholdings endpoint.
+        Uses /manage/institution/holdings/current endpoint.
 
         Args:
-            oclcNumber:             OCLC bibliographic record number; can be an
-                                    integer, or string with or without OCLC # prefix
-            inst:                   registry ID of the institution whose holdings
-                                    are being checked
-            instSymbol:             optional; OCLC symbol of the institution whose
-                                    holdings are being checked
-            response_format:        'application/atom+json' (default) or
-                                    'application/atom+xml'
+            oclcNumbers:            string or list containing one or more OCLC numbers
+                                    to be checked; numbers can be integers or strings
+                                    with or without OCLC # prefix;
+                                    if str, the numbers must be separated by a comma
             hooks:                  Requests library hook system that can be
                                     used for signal event handling, see more at:
                                     https://requests.readthedocs.io/en/master/user/advanced/#event-hooks
 
         Returns:
             `requests.Response` object
         """
-        oclcNumber = verify_oclc_number(oclcNumber)
+        vetted_numbers = verify_oclc_numbers(oclcNumbers)
```

Comment:
What do you think about adding a feature splitting passed OCNs into groups of tens? The API docs say no more than 10 #s per request. You can use `_split_into_legal_volume` function.